src/main/java/faker/generator/FakeOptions.java

```diff
@@ -11,11 +11,5 @@
 @Inherited
 @FakeServiceName(FakeOptionsService.class)
 public @interface FakeOptions {
-    FakeOptionsType value() default FakeOptionsType.YEAR;
-    int minDay() default 1;
-    int maxDay() default 31;
-    FakeOptionsMonths months() default FakeOptionsMonths.JANUARY;
-    int minYear() default 1900;
-    int maxYear() default 2020;
-    String[] days() default {"MONDAY", "TUESDAY", "WEDNESDAY", "THURSDAY", "FRIDAY", "SATURDAY", "SUNDAY"};
+    String[] value();
```

Comment:
Please change for objects, to be more generic

Reply:
As we talked, I changed for String, int and double arrays.